### PR TITLE
[Rust] Added doc comments for Statistics

### PR DESCRIPTION
**Pull request by apepkuss**
This PR is to add the necessary descriptions for the Statistics struct.

### CODE REVIEW

File: bindings/rust/wasmedge-sys/src/statistics.rs [hunk 1]
```diff
@@ -56,3 +72,27 @@ impl Drop for Statistics {
...
+        assert!(result.is_ok());
+        let stat = result.unwrap();
+
+        // check instruction count
+        assert_eq!(stat.instr_count(), 0);
+
+        // check instruction count per second
+        let count = stat.instr_per_sec();
+        assert!(count.is_nan());
+        // assert!(count, 0.0);
```
Comment (hydai) [inline bindings/rust/wasmedge-sys/src/statistics.rs:93]: Instruction per second  = Total executed instructions / total execution time.
If the statistics is not enabled or the total execution time is `0.0`. Then, the instruction per second should be `Nan` which represents something is divided by zero.

**Review by hydai — CHANGES_REQUESTED** [review]

File: bindings/rust/wasmedge-sys/src/statistics.rs (hunk 1 above)
Comment (apepkuss) [reply bindings/rust/wasmedge-sys/src/statistics.rs:93]: For the statement, "If the statistics is not enabled ...", what does "enabled" mean? does it mean the statistics is enabled in a **VM** or something else? Thanks!

Comment (apepkuss) [reply bindings/rust/wasmedge-sys/src/statistics.rs]: The doc comment of the `instr_per_sec` function has been refactored by adding the description on the cases, in which `NaN` is returned. Thanks!

Comment (hydai) [reply bindings/rust/wasmedge-sys/src/statistics.rs]: You can enabled it in the Configure. And the VM/executor will save information to the Statistics module. However, if the VM/executor is not executed before, it should keep empty.

**Review by hydai — APPROVED** [review]

File: bindings/rust/wasmedge-sys/src/statistics.rs (hunk 1 above)
Comment (apepkuss) [reply bindings/rust/wasmedge-sys/src/statistics.rs]: @hydai You're right, `Config` has an option controlling it. Our motivation for this test code is just to verify a corner case, namely, instantiate an individual `Statistics`, and then verify the return values of the APIs of interest. Many thanks for your detailed explanation.
